packages/date-picker/src/vaadin-month-calendar.js

```diff
@@ -401,6 +402,27 @@ class MonthCalendar extends ThemableMixin(PolymerElement) {
     }
   }
 
+  __getDayAriaLabel(date) {
```

Comment:
The old aria-label formatting logic makes some bad assumptions on how dates are expressed in different languages ("date month year weekday" regardless of the language). Why not use Intl (this can surely be done separately)?

Reply:
Created #3383 to discuss `Intl.DateTimeFormat`. Let's keep the current logic for now.

Reply:
That issue seems to be about a larger refactor (replacing datepicker.i18n values with those derived from Intl)

I'm suggesting to change the aria-labels of the month calendar cells only without refactoring other parts of i18n. aria-label only affects AT and the preferred language in that case is probably always the one in use in the browser (may be different from date picker's i18n). This would be aligned with how the native `<input type="date">` dropdown cells are announced.

Reply:
Good point. Created another issue #3385 - that one we can probably do in Vaadin 23 (marked as refactor).